test/integration/backward_compatible/serial/ListenersOnReconnectTest.js

```diff
@@ -67,14 +64,32 @@ describe('ListenersOnReconnectTest', function () {
                     }
                 }
             };
-            await map.addEntryListener(listener, 'keyx', true);
+            const registrationId = await map.addEntryListener(listener, 'keyx', true);
 
             await RC.terminateMember(cluster.id, member.uuid);
+            // Assert that the connection is closed and the listener is removed.
+            await TestUtil.assertTrueEventually(async () => {
+                const activeConnections = TestUtil.getConnections(client);
+                expect(activeConnections.length).to.be.equal(0);
+
+                const activeRegistrations = TestUtil.getActiveRegistrations(client, registrationId);
+                const connectionsThatHasListener = [...activeRegistrations.keys()];
+                expect(connectionsThatHasListener.length).to.be.equal(0);
```

Comment:
```suggestion
                expect(activeRegistrations.length).to.be.equal(0);
```
or size? I don't remember the exact name

Reply:
Ok, done.